Consul/Interfaces/IPolicyEndpoint.cs

```diff
@@ -37,5 +37,7 @@ public interface IPolicyEndpoint
         Task<QueryResult<PolicyEntry>> Read(string id, QueryOptions q, CancellationToken ct = default);
         Task<WriteResult<PolicyEntry>> Update(PolicyEntry policy, CancellationToken ct = default);
         Task<WriteResult<PolicyEntry>> Update(PolicyEntry policy, WriteOptions q, CancellationToken ct = default);
+        Task<QueryResult<TemplatedPolicyResponse>> ReadATemplatedPolicyByName(string name, QueryOptions q, CancellationToken ct = default);
```

Comment:
Including the `A` article in the name makes it sound a little bit awkward. Let's just call it `ReadTemplatedPolicyByName`